jme3-core/src/main/java/com/jme3/material/RenderState.java

```diff
@@ -480,6 +480,10 @@ public enum StencilOperation {
     StencilOperation backStencilDepthPassOperation = StencilOperation.Keep;
     TestFunction frontStencilFunction = TestFunction.Always;
     TestFunction backStencilFunction = TestFunction.Always;
+    int frontStencilReference=0;
+    int backStencilReference=0;
+    int frontStencilMask=Integer.MAX_VALUE;
+    int backStencilMask=Integer.MAX_VALUE;

```

Comment:
Please fix the formatting issue. (Add space before and after "=")

Reply:
done